wallet-new/integration/Util.hs

```diff
@@ -170,6 +171,22 @@ noLongerThan action maxWaitingTime = do
         Left _  -> return Nothing
         Right a -> return $ Just a
 
+confirmTransaction
+    :: [Transaction]
+    -> Transaction
+    -> PropertyM IO ()
+confirmTransaction transactions txn = do
+    let txnEntry: _ = filter ( \x -> (txId x) == (txId txn)) transactions
+    log $ "Resp   : " <> ppShowT txnEntry
+    liftIO $ [txStatus txnEntry] `shouldContain` [InNewestBlocks, Persisted]
+
+
+log :: MonadIO m => Text -> m ()
+log = putStrLn . mappend "[TEST-LOG] "
+
+ppShowT :: Show a => a -> Text
+ppShowT = fromString . ppShow
```

Comment:
If those are only used in the above function, let's make them part of a `where` clause. There's no reason why they should be available for the rest.

Reply:
Made changes as suggested, commits squashed. `log`, `ppShowT` are also used in `wallet-new/integration/RandomStateWalk.hs`

Reply:
Okay, we gotta remove this file at some point :| ... But not part of this PR.

Thanks @paweljakubas waiting for the CI and merging if everything's okay.